fixtures/ed/gtfs_google_example/routes.txt

```diff
@@ -1,6 +1,6 @@
 route_id,agency_id,route_short_name,route_long_name,route_desc,route_type,route_url,route_color,route_text_color
-AB,DTA,10,Airport - Bullfrog,,3,,,
+AB,DTA,10,Airport - Bullfrog,,3,,,FFD700
```

Comment:
hum this directory is the raw google GTFS example, I think it's better if we do not change it.

Maybe you can change another GTFS fixture ?